processing/src/main/java/org/apache/druid/query/groupby/strategy/GroupByStrategyV2.java

```diff
@@ -442,8 +425,8 @@ public boolean doMergeResults(final GroupByQuery query)
         }
 
         GroupByQuery subtotalQuery = baseSubtotalQuery
-            .withLimitSpec(subtotalQueryLimitSpec)
-            .withDimensionSpecs(newDimensions);
+            .withLimitSpec(subtotalQueryLimitSpec);
+            //.withDimensionSpecs(newDimensions);
```

Comment:
Please don't include commented-out code.

Reply:
Yup. I had removed it on my local but forgot to push.